doc/bibliography.bib

```diff
@@ -832,31 +839,31 @@ @Article{rohm12a
   journal = {European Physical Journal Special Topics},
   year = {2012},
   volume = {210},
-  doi = {10.1140/epjst/e2012-01639-6},
+  number = {1},
   pages = {89--100},
+  doi = {10.1140/epjst/e2012-01639-6},
 }
 
-@BOOK{rubinstein03a,
+@Book{rubinstein03a,
   title = {Polymer Physics},
   publisher = {Oxford University Press},
   year = {2003},
   author = {Michael Rubinstein and Ralph H. Colby},
   address = {Oxford, UK},
 }
 
-@ARTICLE{schatzel88a,
-  author = {Sch\"{a}tzel, K. and Drewel, M. and Stimac, S},
-  title = {Photon Correlation Measurements at Large Lag Times: Improving Statistical Accuracy},
+@Article{schatzel88a,
+  author = {Sch\"{a}tzel, Klaus and Drewel, Martin and Stimac, Sven},
+  title = {Photon correlation Measurements at Large Lag Times: Improving Statistical Accuracy},
```

Comment:
Why small `c'? Screws up title case (which also is used for formatting the title on on the referenced webpage via doi)